amr-wind/equation_systems/icns/source_terms/DragForcing.cpp

```diff
@@ -184,6 +184,7 @@ void DragForcing::operator()(
     const auto tiny = std::numeric_limits<amrex::Real>::epsilon();
```

Comment:
you can remove all tiny uses and just use `constants::EPS`. Or any of the other constants we provide. In this case, `SMALL_NUM` might work too. Check out `constants.H`.